{"uuid":"61629fe3-be8c-497b-bbd9-c6b8c7471c49","historyId":"2949a9c9645bb9f520e79d0eb5bde852","fullName":"tests.LoginTest.loginTest","labels":[{"name":"package","value":"tests.LoginTest"},{"name":"testClass","value":"tests.LoginTest"},{"name":"testMethod","value":"loginTest"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"tests.LoginTest"},{"name":"host","value":"Ujjawals-MacBook-Air.local"},{"name":"thread","value":"21053@Ujjawals-MacBook-Air.local.main(3)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginTest","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.RuntimeException: Error reading excel file: src/main/resources/TestCases_Login_Register.xlsx","trace":"java.lang.RuntimeException: java.lang.RuntimeException: Error reading excel file: src/main/resources/TestCases_Login_Register.xlsx\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:803)\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:728)\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:911)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1604)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\n\tat org.testng.TestNG.run(TestNG.java:1082)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:102)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:91)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\nCaused by: java.lang.RuntimeException: Error reading excel file: src/main/resources/TestCases_Login_Register.xlsx\n\tat utils.ExcelUtils.getSheetData(ExcelUtils.java:45)\n\tat tests.LoginTest.getLoginData(LoginTest.java:14)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\n\t... 30 more\nCaused by: java.io.FileNotFoundException: src/main/resources/TestCases_Login_Register.xlsx (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:185)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:139)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:109)\n\tat utils.ExcelUtils.getSheetData(ExcelUtils.java:21)\n\t... 36 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1759658264421,"stop":1759658264425}